src/features/RecipeLibrary/data/queries.ts

```diff
@@ -17,6 +17,8 @@ export const SEARCH_RECIPES = gql(`
                             id
                             imageUrl
                             name
+                            email
+                            provider
```

Comment:
`Q:` Why do we need provider for the library view?

Reply:
Ah! We do not. This was codegen science that accidentally made it's way into the PR. Thank you for noticing.